### PR TITLE
Pop up branch

### PR DESCRIPTION
**Pull request by sulaimonaa**
Add pop-up modal to work done
Add styling to the modal
Add dynamic scripting

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -316,6 +293,172 @@
     border: 1px solid #344563;
   }
 
+  /* Detail Section Starts Here */
+
+  .detail-card {
```
Comment (BregornOriginal) [inline style.css:305]: - Kindly make your project matching with the Figma design, fix the size of the card 

- Kindly correct the position of the elements, as you can see on the Figma design, the text is on the top and bottom in the bottom


Your design

image


Figma design

image

File: style.css [hunk 2]
```diff
@@ -602,6 +745,24 @@
     gap: 0;
   }
 
+  .detail-card-body-left {
+    width: 70%;
+  }
+
```
Comment (BregornOriginal) [inline style.css:769]: On the desktop version, when I open the project detail, the popup window should be on the center of the screen with the scroll locked and the background with some `filter: blur`

- Kindly position your card detail in the middle of the screen

- Kindly add the blur filter on the background of the popup window

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @sulaimonaa and @Ridalord  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (5)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ README looks professional
✅ Good work implementing the JavaScript section 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -316,6 +293,181 @@
...
+  .blur-modal {
+    filter: blur(3px);
+  }
+
+  .detail-card {
+    display: none;
+  }
+
+  .detail-card--show {
```
Comment (christianbiring1) [inline style.css:309]: - [ ] Kindly specified in the previous review, Kindly make sure your popup modal is centered from the top when it is opened.
You may consider fixing this by readjusting your `margin`. you may consider removing the auto and setting it with a specific value.
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @sulaimonaa , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STAUS: REQUEST CHANGES :recycle: 
soclose


 ## Highlights
- [ ] Great work on readjusting your mobile styles on the popup and set it correctly as it is in the original design :+1: 
- [ ] All linters cheks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and ``
summary :+1: 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (Qoosim) [inline style.css:309]: - [ ] Great job implementing the required changes from the previous reviewer. However, there're few things you still need to work on before your project completely meet the project requirement.
  - Your popup window must contain all the information as we have them on the Figma design. You should be able to style your popup card such that it contains all the information on the card as they're on the figma design. At thi moment, your card does not display all the information. Kindly address that.
  - The layout on your design must match the Figma design.

### Your Design
your-popup-desktop

### Figma Design

tem-popup-desktop

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**